### PR TITLE
Add Search API

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: examples/search/main.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	ctx := context.Background()
+
+	// Send a bunch of Sparql query and print out its response.
+	qStr := "Santa Clara"
+
+	r, err := c.Search(ctx, &pb.SearchRequest{Query: qStr})
+	if err != nil {
+		log.Fatalf("could not Search: %v", err)
+	}
+	log.Printf("Search: %v", r.GetSection())
```
Comment (pradh) [inline examples/search/main.go]: Given as this is the only test we have for this, add some sanity checks?

Also add an invocation to run_all.sh?

File: store/search.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+	"google.golang.org/api/iterator"
+)
+
+func (s *store) Search(
+	ctx context.Context, in *pb.SearchRequest, out *pb.SearchResponse) error {
+	result := map[string]*pb.SearchResultSection{}
+	tokens := strings.Split(strings.ToLower(in.GetQuery()), " ")
+	qStr := fmt.Sprintf(
+		"SELECT id, type, extended_name FROM `%s`.Instance WHERE type != \"CensusTract\"", s.bqDb)
```
Comment (pradh) [inline store/search.go]: PowerPlant and PowerPlantUnit are pretty spammy too.

https://browser.datacommons.org/search?query=Mountain+View

File: proto/mixer.proto [hunk 3]
```diff
@@ -252,6 +252,43 @@ message GetPlacesInResponse {
...
+// Search request sent to the mixer.
+message SearchRequest {
+  // Search query which is just a plain text string, e.g. "california".
+  // This is not a datalog query, but a plain text query. The mixer is free
+  // to choose an interpretation of the query, e.g. using NLP or just plain
+  // keyword search and return relevant entities from the graph.
+  string query = 1;
+
+  // Maximum number of entities to return, default to 200.
+  int32 max_results = 2;
```
Comment (pradh) [inline proto/mixer.proto]: This is maximum across all results? s.t. if total results exceed 200, then they are not reliable?

File: examples/search/main.go (hunk 1 above)
Comment (shifucun) [reply examples/search/main.go]: done

**Pull request merged**